schema.graphql

```diff
@@ -296,3 +296,65 @@ type IssuerIdentityWithClaims @entity {
   did: String! @index
   claims: [ClaimJson!]!
 }
+
+type AssetHolder @jsonField {
+  did: String!
+  amount: String!
+}
+
+type AssetDocument @jsonField {
+  id: Int! # document id
+  name: String!
+  link: String!
+}
+
+type AssetIdentifier @jsonField {
+  type: String!
+  value: String!
+}
+
+type AssetPendingOwnership @entity {
```

Comment:
(nit) 

I feel like this would be clearer with the word Transfer even if its a bit long. I think using Auth instead of pending works to, since its a core concepts. e.g.`AssetOwnershipTransferAuth`

Reply:
I initially called it auths but since its use case is very narrow now, I called it what it's being used for. The thing is when transferring asset ownership, 2 types of auths are created, ownership transfer and become agent. I also added the word pending to indicate it's a temporary state.
How about `AssetPendingOwnershipTransfer`?

Reply:
I like `AssetPendingOwnershipTransfer`